### ISSUE TITLE
Set aria-expanded attribute to match drawer/notify visibility

### ISSUE THREAD

**Issue by kirsty-hames** [cgkineo/adapt-visua11y#96]
### Subject of the issue
As per https://github.com/adaptlearning/adapt-contrib-core/issues/409, buttons and links that unfold / collapse content on the page are to be coded with aria-expanded attribute (according to _success criteria 4.1.2 Name, Role, Value_).

### Expected behaviour
`.nav__visua11y-btn` should have the relevant `aria-expanded` attribute applied to match drawer visibility.

### Actual behaviour
No such attribute exists.
